packages/react-components/react-badge/src/components/PresenceBadge/PresenceBadge.tsx

```diff
@@ -4,14 +4,19 @@ import { usePresenceBadgeStyles_unstable } from './usePresenceBadgeStyles';
 import { renderBadge_unstable } from '../../Badge';
 import type { PresenceBadgeProps } from './PresenceBadge.types';
 import type { ForwardRefComponent } from '@fluentui/react-utilities';
+import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
```

Comment:
_nit:_ Keep `import type` statements at the bottom

```suggestion
import { useCustomStyleHooks_unstable } from '@fluentui/react-shared-contexts';
import { renderBadge_unstable } from '../../Badge';
import type { PresenceBadgeProps } from './PresenceBadge.types';
import type { ForwardRefComponent } from '@fluentui/react-utilities';
```